### ISSUE TITLE
Multi tenancy support

### ISSUE THREAD

**Issue by MarcelHoek** [rubrikinc/rubrik-sdk-for-python#21]
Hi,

it seems that the current python SDK does not support organizations. So if a user is part of the global **and** annother organization the python SDK does not support this and throws an error 403.

Basically the python SDK should add the organization to the API request, like below:
api/v1/session?organization_id=Organization:::a7d40b13-b368-4af8-a67d-e32c75ff1cb5

Can this organization support be added to the module?

Marcel

**Comment by drew-russell** [comment]
You're absolutely correct. The SDK currently does not support Organizations. We will work on adding this and get it added ASAP for you.

**Comment by MarcelHoek** [comment]
Thanks. No rush from our side. We have a workaround for it (created another user which is only member of the global tenant).
